modules/vulnerabilities/unix/http/linuxki_rce/secgen_metadata.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<vulnerability xmlns="http://www.github/cliffe/SecGen/vulnerability"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.github/cliffe/SecGen/vulnerability">
+
+  <name>LinuxKI Toolset 6.01 Remote Command Execution</name>
+  <author>James Davis</author>
+  <module_license>MIT</module_license>
+  <description>
+    This module exploits a vulnerability in LinuxKI Toolset 6.01 and below which allows
+    remote code execution.
+    The kivis.php pid parameter received from the user is sent to the shell_exec function,
+    resulting in security vulnerability.
+  </description>
+
+  <type>web</type>
+  <type>in_the_wild</type>
+  <privilege>user_rwx</privilege>
+  <access>remote</access>
+  <platform>linux</platform>
+  <difficulty>low</difficulty>
+
+  <read_fact>port</read_fact>
+  <read_fact>strings_to_leak</read_fact>
+  <read_fact>leaked_filenames</read_fact>
+
+  <default_input into="port">
+    <generator module_path=".*/random_unregistered_port" />
+  </default_input>
+
+  <!-- flags or other secrets exposed after exploitation -->
+  <default_input into="strings_to_leak">
+    <generator type="flag_generator" />
```

Comment:
Use a message generator instead of a flag_generator, for non-CTF scenarios. When it's needed, a flag will be passed in by the scenario.